### PR TITLE
chore: update copy on iOS notifications saved search flow and profile 

### PR DESCRIPTION
**Pull request by gkartalis**
The type of this PR is: **Enhancement**

This PR resolves [FX-3073]

### Description

- copy changes on iOS notifications saved search flow
- copy changes on Profile > PushNotifications screen

### Screenshots

|PushAuthorizationStatus.Denied|PushAuthorizationStatus.NotDetermined|
|---|---|
|IMG_4488|IMG_4478|
|IMG_4487|IMG_4486|

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

<details><summary>Changelog updates</summary>

#### iOS user-facing changes

- copy changes on iOS notifications saved search flow
- copy changes on Profile PushNotifications

</details>

[FX-3073]: https://artsyproduct.atlassian.net/browse/FX-3073

### CODE REVIEW

File: src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx [hunk 1]
```diff
@@ -144,8 +144,8 @@ export const SavedSearchBanner: React.FC<SavedSearchBannerProps> = ({
           return createSavedSearch()
         case PushAuthorizationStatus.Denied:
           return Alert.alert(
-            "Turn on notifications",
-            'To receive push notification alerts from Artsy on new works by this artist, you\'ll need to enable them in your iOS Settings. Tap Notifications, and then toggle "Allow Notifications" on.',
+            "Artsy would like to send you Notifications",
```
Comment (dblandin) [inline src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx:147]: **suggestion**: Based on Jonathan's [comment in Jira](https://artsyproduct.atlassian.net/browse/FX-3073?focusedCommentId=39184), I think "notifications" should be lowercase. Will double-check in Slack.

**Review by dblandin — COMMENTED** [review]
Copy changes look good! Left one suggestion for the casing of "notifications" in the alert title and profile screen.

File: src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx (hunk 1 above)
Comment (gkartalis) [reply src/lib/Components/Artist/ArtistArtworks/SavedSearchBanner.tsx:147]: perfect! After clearing this out in [slack](https://artsy.slack.com/archives/C9SATFLUU/p1625744360158600?thread_ts=1625732197.149000&cid=C9SATFLUU) I will go ahead and change them to lowercase

**Review by mdole — APPROVED** [review]
Nice! Let's 🚢

**Review by dblandin — APPROVED** [review]

**Pull request merged**